App/TopTrumps/TopTrumps.Data/Services/UsersService.cs

```diff
@@ -287,5 +289,27 @@ public async Task<IList<User>> GetUsersInRoleAsync(string roleName, Cancellation
                 cancellationToken,
                 new { normalizedName = roleName.ToUpper() })).ToList();
         }
+
+        public Task<User> GetUser(int id)
+        {
+            throw new NotImplementedException();
+        }
+
+        public Task<IEnumerable<User>> GetUsers()
+        {
+            throw new NotImplementedException();
+        }
+
+        public async Task<IEnumerable<User>> GetAllUsers()
+        {
+            return await _repo.QueryAsync<User>(
+                _helper.GetAllUsersQuery(),
+                new CancellationToken());
+        }
+
+        Task<User> IUsersService.GetAllUsers()
+        {
+            throw new NotImplementedException();
+        }
```

Comment:
Not sure what this is for?

Reply:
Thought this was needed to implement the interface for IUsersService